frontend/components/card/discussion/CardDiscussionText.vue

```diff
@@ -35,6 +36,6 @@ import type { DiscussionText } from "~/types/card-discussion-text";
 
 defineProps<{
   isPrivate?: boolean;
-  discussionText: DiscussionText;
+  discussionTexts: DiscussionText;
```

Comment:
I'm gonna revert this back as we'll be querying from the backend for this, @UnknownSean8 :) Also, the type here would need to be `DiscussionText[]`.